src/pages/wallet/add.custom.token.confirm.js

```diff
@@ -137,6 +138,14 @@ class AddCustomToken extends Component {
         });
       } catch (error) {
         const notification = getErrorNotification(error.code) || getDefaultErrorNotification();
+
+        if (!getErrorNotification(error.code)) {
```

Comment:
I might be missing something, but shouldn't we report this error unconditionally?

Reply:
I partially agree. The problem is that sometimes "errors" are "insufficient funds", that are not important to us, and it would spam the service.

Reply:
Honestly, I am unsure what errors are "important" and which are useless. 

The initial group of reportings is at best a guess. In most cases, I have no idea when they are triggered. I think going forward, the error report content should be analyzed and if found to be useless, remove the from the code.